src/App.js

```diff
@@ -50,9 +57,31 @@ const BodyComponent=()=>{
 const AppLayout =()=>(
     <>
     <HeadingComponent/>
-    <BodyComponent/>
+    <Outlet/>
     </>
 );
 
+const appRouter = createBrowserRouter([
+    {
+        path : "/",
+        element : <AppLayout/>,
+        errorElement : <ErrorComponent/>,
+        children : [
+            {
```

Comment:
You can add another child with index: true property to display some components by default in oulet.

Reply:
With Assignment 8 I will push that code